src/test/java/de/tum/in/www1/artemis/AttachmentUnitIntegrationTest.java

```diff
@@ -16,6 +16,7 @@
 import de.tum.in.www1.artemis.repository.*;
 import de.tum.in.www1.artemis.util.ModelFactory;
 
+@Deprecated // Moved to Lecture microservice. To be removed
 public class AttachmentUnitIntegrationTest extends AbstractSpringIntegrationBambooBitbucketJiraTest {
```

Comment:
Why did you remove the section in `createAttachmentUnit_asInstructor_shouldCreateAttachmentUnit()` without replacement?

Reply:
The removed section was related to calling /api/attachments endpoint which is not part of the lecture microservice. At first I thought that this can be removed as we want to test the attachmentunit. Now, that I had a second look the creation of the attachment is important in order to test the link between the attachment and attachment unit. So I added it back. But since we cannot call the endpoint I call the attachment repository